bin/deploy

```diff
@@ -34,6 +34,17 @@ test -f "$LSSTSW/anaconda/.deployed" || ( # Anaconda
     echo "::: Deploying Anaconda ${ANACONDA_VERSION} for ${ana_platform}"
     curl -# -L -O http://repo.continuum.io/archive/${ana_file_name}
     bash ${ana_file_name} -b -p "$LSSTSW/anaconda"
+
```

Comment:
No -- we want to keep it to 2.1.0 (2.2.0 forces an upgrade to an incompatible version of IPython -- which users will hate), and just update the 'system' conda package (that itself resolves the libm bug).